python/lsst/ts/observatory/control/base_tcs.py

```diff
@@ -36,6 +36,24 @@
 
 class BaseTCS(RemoteGroup, metaclass=abc.ABCMeta):
     """Base class for Telescope Control System.
+
+    Parameters
+    ----------
+    components : `list` [`str`]
+        A list of strings with the names of the SAL components that are part
+        of the telescope control system group.
+    domain : `lsst.ts.salobj.Domain`
+        Domain for remotes. If `None` create a domain.
+    log : `logging.Logger`
+        Optional logging class to be used for logging operations. If `None`,
+        creates a new logger. Useful to use in salobj.BaseScript and allow
+        logging in the class use the script logging.
```

Comment:
Same comment as before. In line 46 you use "create" and in line 49 "creates". Please choose one and stick to that. Also, I don't understand the last sentence.